docs/articles/documentation/test-api/intercepting-http-requests/mocking-http-responses.md

```diff
@@ -12,6 +12,8 @@ Mocking is useful when the tested app uses infrastructure that is difficult to d
 * [The onRequestTo Method](#the-onrequestto-method)
 * [The respond Method](#the-respond-method)
   * [A Custom Response Function](#a-custom-response-function)
+* [Examples](#examples)
+  * [Mocking Cross-Domain Responses](#mocking-cross-domain-responses)
```

Comment:
`[Mocking Cross-Domain Requests](#mocking-cross-domain-requests)`

Reply:
well, the topic is called `Mocking HTTP Responses` so it says we're mocking responses not requests.

Reply:
Is the topic title wrong?

Reply:
As I know, for this case the phrase `Mocking http requests` is more used than `Mocking http responses'.
See module descriptions in https://www.npmjs.com/search?q=mocking%20requests.